mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -891,6 +891,10 @@ The `workloads_config` block supports:
   (Optional)
   Configuration for resources used by Airflow workers.
 
+* `triggerer` -
```

Comment:
move between scheduler and web_server part

Reply:
Add information it is in beta (`(Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)`)